zoraActions/.htaccess

```diff
@@ -0,0 +1,5 @@
+Header set Access-Control-Allow-Origin *
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^$ https://github.com/Fspiga13/zoraActionsOntology [R=302,L]
+RewriteRule ^(.+)$ https://github.com/Fspiga13/zoraActionsOntology/$1 [R=302,L]
```

Comment:
Did you mean to point to the raw content?  The above will go to the github UI for a resource.

Reply:
Yes, I think it's better to point to the raw content.

Reply:
Then please update your PR and we'll merge it :)